src/bucket/complete_multipart_upload.gleam

```diff
@@ -0,0 +1,101 @@
+import bucket.{type BucketError, type Credentials}
```

Comment:
Add a link to the documentation as a module doc comment for each module please

Reply:
done in e1ce3d0